condor_templates.go

```diff
@@ -169,10 +176,36 @@ k8s:
     get-analysis-id:
         header: {{.GetString "k8s.get-analysis-id.header"}}`
 
+// CondorBytes formats a number of bytes to a condor format (rounding up to the nearest KiB until it's at least 1MiB, then rounding up to the nearest MiB)
+func CondorBytes(bytes int64) string {
+	if bytes < 1024 {
+		return "1KB"
+	}
+
+	if bytes < 1048576 {
+		kb := bytes / 1024
+		if bytes%1024 > 0 {
+			kb = kb + 1
+		}
+		return fmt.Sprintf("%dKB", kb)
+	}
+
+	mb := bytes / 1048576
+	if bytes%1048576 > 0 {
+		mb = mb + 1
+	}
+
+	return fmt.Sprintf("%dMB", mb)
+}
+
```

Comment:
I'm kind of picking nits here, but it would be kind of nice to have constants for the number of bytes per kibibyte and the number of bytes for mebibyte. If I were modifying this code, I can easily imagine myself mistyping one of the numbers (well, probably not the number of bytes per kibibyte) and introducing a subtle bug.

Reply:
Cool. I always feel bad for requesting stuff like this, but it does help in some cases.